src/main/java/PDLayer.mdl

```diff
@@ -570,6 +570,8 @@ type PDAnnot extends PDObject {
 	property Alt: String;
 	% true, if annotation rectangle is outside page CropBox
 	property isOutsideCropBox: Boolean;
+	% true if this annotation has an ancestor struct elem with the standard type 'Artifact'
```

Comment:
% true if this annotation is contained in an Artifact structure element (directly or indirectly)